public/assets/scss/player/_player.scss

```diff
@@ -174,6 +174,15 @@
 			margin: 0;
 			cursor: pointer;
 		}
+
+		.error-el{
+			top: 50%;
+			margin-top: -2em;
+			background: $bgAccentColor;
```

Comment:
Name of variable `bgAccentColor` should be written in all lowercase letters with hyphens instead of underscores